pkg/docker/storage.jsx

```diff
@@ -0,0 +1,511 @@
+/*
+ * This file is part of Cockpit.
+ *
+ * Copyright (C) 2015 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+define([
+    "jquery",
+    "react",
+    "base1/cockpit",
+    "storaged-client/client",
+    "storaged-client/utils",
+    "performance/dialog-view"
+], function($, React, cockpit, storage_client, storage_utils, dialog_view) {
+    var _ = cockpit.gettext;
+    var C_ = cockpit.gettext;
+
+    function init_model(callback) {
```

Comment:
You don't have to change what is already written, but note that we decided to use camelCase in new js code to match what other projects / React is doing.